agent/src/agent/tools.py

```diff
@@ -92,3 +88,42 @@ def sdc_record_map_to_dict(record: dict):
         return datetime.fromtimestamp(record['value'] // 1000).strftime('%Y-%m-%d %H:%M:%S')
 
     return record['value']
+
+
+def get_influx_client(host, username=None, password=None, db=None) -> InfluxDBClient:
```

Comment:
get_influx_client, has_write_access and get_mongo_client should be somewhere in the source package